packages/widgets/src/menubar.ts

```diff
@@ -591,6 +600,17 @@ export class MenuBar extends Widget {
     }
   }
 
+  /**
+   * Handle the `'focus'` event for the menu bar.
+   */
+  private _evtFocus(event: FocusEvent): void {
```

Comment:
Why do we wish to reset the active menu item when the menu bar receives focus?

I would like to make the Lumino menubar behave more like the [example menubar in the ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/example-index/menubar/menubar-navigation).

Reply:
This is caused by the interpretation of what it means for the "menubar to receive focus", as I put in the main reply. I'll switch to the interpretation used in the example, which will also make the code much simpler.